docs/astria.md

```diff
@@ -0,0 +1,145 @@
+# [Astria](https://www.astria.org/): Decentralized by Default
+ - [The Astria dev-cluster](#the-astria-dev-cluster)
+ - [Architecture](#architecture)
+    - [Users](#users)
+    - [Rollups](#rollups)
+    - [Composer (under construction)](#composer-under-construction)
+    - [The Shared Sequencer](#the-shared-sequencer)
+    - [Relayer](#relayer)
+    - [Data Availability](#data-availability)
+    - [Conductor](#conductor)
+ - [Running the Cluster](#running-the-cluster)
+    - [Installation](#installation)
+    - [Deployment](#deployment)
+    - [Faucet and Block Explorer](#faucet-block-explorer-and-test-data)
+    - [Multiple Rollups](#run-multiple-rollups)
+    - [What's Going on Under the Hood?](#whats-going-on-under-the-hood)
+ - [Summary](#summary)
+
+## The Astria [dev-cluster](https://github.com/astriaorg/dev-cluster)
+The Astria Shared Sequencer Network replaces centralized sequencers, allowing many rollups to share a single decentralized network of sequencers that’s simple and permissionless to join. This shared sequencer network provides out-of-the-box censorship resistance, fast block confirmations, and atomic cross-rollup composability – all while retaining each rollup’s sovereignty.
+
+The Astria dev-cluster is the collective stack of all of Astria's components packaged together using Kubernetes. Although we refer to Astria as just a shared sequencer, it is composed of many smaller components that each fulfill a specific function to enable the final shared sequencing functionality. Instead of users and developers having to keep track of and manage all the individual components, the dev-cluster is built to make locally deploying and testing Astria as simple and robust as possible. 
+
+This document outlines what each of the components do, and provides some basic instructions for running the dev cluster yourself.
+
+## Architecture
+If you have seen any of [Josh's](https://twitter.com/Jskybowen) recent presentations about Astria, you will likely have seen this image:
+![Astria at a high level](images/astria-high-level.png)
+This shows the basic structure of what a shared sequencer is but purposely doesn't dive into any of the details of Astria to keep things simple. However, Astria's general architecture does break down into the basic items shown above.
+
+When expanded to show more detail, Astria's architecture looks like this:
+![Astria Architecture](images/astria-architecture.png)
```

Comment:
This diagram is great. I think we should rename "namespaced transactions" to "rollup transactions" but I'm open to bikeshedding.

Reply:
I also don't think the think that conductor retrieves from DA/Celestia should be called a "block bundle transaction". Jordan's sequencer diagram calls it "Batch Blocks" which isn't great either...

Reply:
On verbiage from DA.

From celestia POV: We are getting "namespaced blobs". I agree namespaced isn't the best term for us here though.

Each namespaced blob either contains sequencer header data or rollup transactions. The pair makes what we have previously called a "rollup transaction batch", but each celestia block has many of these, essentially a batch of batches? Kinda confusing. I had used "Block batches" out of laziness, it's not great.

What if we call the set of sequencer header + tx set an "Unexecuted Rollup Block" (URB as shorthand). URBs are deterministically derived from sequencer blocks, and are deterministically executed into Rollup Blocks. We utilize them to minimize data bandwidth (we could just be using full sequencer blocks theoretically), and are verifiable as belonging to the sequencer block via commitments (work in progress). 

What conductor gets from Celestia is then an "unexecuted rollup block batch" or a "URB Batch".

Reply:
I changed this to "unexecuted rollup block" and "URB batch" for now.